ui/app/routes/vault/cluster/settings/auth/configure/section.js

```diff
@@ -86,7 +86,7 @@ export default Route.extend(UnloadModelRoute, {
         // if you haven't saved a config, the API 404s, so create one here to edit and return it
         if (e.httpStatus === 404) {
           config = this.store.createRecord(modelType, {
-            id: backend.id,
```

Comment:
I can't find where, but _somewhere_ a record is being created by ember-data (that doesn't persist to the server) which caused this error
```
router.js:1129 Error while processing route: vault.cluster.settings.auth.configure.section 
The id oidcNEW has already been used with another 'auth-config/oidc' record.
 Error: The id oidcNEW has already been used with another 'auth-config/oidc' record.
```

![image](https://github.com/user-attachments/assets/18a30eda-1525-4f49-9cf6-80912531743d)